UncertainSCI/ttr/compute_ttr.py

```diff
@@ -0,0 +1,672 @@
+import numpy as np
+import scipy.special as sp
+
+from UncertainSCI.opoly1d import eval_driver, \
+        leading_coefficient_driver, gauss_quadrature_driver
+
+from UncertainSCI.utils.compute_subintervals import compute_subintervals
+from UncertainSCI.utils.quad import gq_modification_composite, \
+        gq_modification_unbounded_composite
+from UncertainSCI.utils.array_unique import array_unique
+
+"""
+Predictor-corrector method
+"""
+def predict_correct_bounded(a, b, weight, N, singularity_list, Nquad=10):
```

Comment:
Are there tests for any of these? I couldn't find anything. Even a `git grep lanczos` only popped up the definitions in this file, but didn't show anything else, i.e., nothing is calling these functions. Can you write some basic tests? Even just sanity checks are fine for now.

Reply:
A quick test for PC and orthogonality was written in test_ttr.py.